upstream-community-operators/kiali/kiali.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: kiali
+channels:
+- name: alpha
+  currentCSV: kiali.v0.1.0
```

Comment:
This should match the name of the CSV (in the `metadata.name` section of the CSV)

Reply:
fixed. this is now `kiali-operator.v0.1.0`

When the "real" stuff is published, this version string will be different. This "v0.1.0" is just a placeholder for my initial operator work (though I do have v0.1.0 of the operator published on quay.io, but that is just my local build).